libcxx/include/__vector/vector.h

```diff
@@ -1128,6 +1128,21 @@ vector<_Tp, _Allocator>::__emplace_back_slow_path(_Args&&... __args) {
   return this->__end_;
 }
 
+template <class _If, class _Else>
+_LIBCPP_HIDE_FROM_ABI _LIBCPP_CONSTEXPR_SINCE_CXX14 void __if_likely_else(bool __cond, _If __if, _Else __else) {
```

Comment:
This needs a comment explaining what's going on and why it's required. Also, a link to a LLVM bug report.